src/test/java/com/scottlogic/utils/UserPostFilterData.java

```diff
@@ -69,4 +70,118 @@
     public static List<UserPost> getFiltered_authorMatt_OR_keywordfunky() {
         return Arrays.asList(userPost2, userPost5, userPost6);
     }
+
+    public static List<UserPost> getFiltered_authorMatt_OR_likedPosts() {
+        return Arrays.asList(userPost2, userPost1, userPost3, userPost4, userPost6);
+    }
```

Comment:
Do these util methods get used in more than 1 test? If so I think keep them - we don't like duplication.

However, if they are used for only one test, why not just have say `List<UserPost> filteredPosts = Arrays.asList(userPost2, userPost1, userPost3, userPost4, userPost6);` in the test themselves.

Upsides would be:
- Readable. I know exactly what my expectations are for this test.
- Stops this file getting very very big when we add more and more unit tests.

What are your thoughts? Are there any obvious downsides?

Reply:
Having all these UserPosts at the top of each test class was a bit of an eyesore, and instead of settings getters for each UserPost I created methods to get the test result. I do agree with your points and I don't expect the methods to get used again, although Dillon mentioned it was more down to preference here so I went with this approach - is your approach described here used more?